datasets/mapping.yml

```diff
@@ -81,3 +81,14 @@ Requirements:
             table: Funding_Request__c
         Primary_Contact__c:
             table: Contact
+Reviews:
+    sf_object: Review__c
+    fields:
+        - Name
+        - Comments__c
+        - DueDate__c
```

Comment:
@ClayTomerlin , do you recommend we use [relative dates](https://cumulusci.readthedocs.io/en/stable/data.html?highlight=relative%20date#relative-dates) for any part of the OFM dataset?

Reply:
@spelak-salesforce I do! But I created [W-9200006 | OFM - Update storytelling data](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000KSxKYAW/view) and included notes (plus an AC) to consider this. I think it would be best to implement it with an update across multiple records as opposed to just this table when the Funding Programs and Funding Requests are outdated as well. Let me know what you think!

Reply:
This Pull Request is a great opportunity to get this change.

Reply:
Yeah, I added enough to be able to verify and test this feature. I agree with the need to constantly add to it where we can, but I think theres enough to consider and change that working with Docs and Product to get an idea of what an overhaul would include is important. Appreciate the prioritization of it for sure!